RELEASE-PROCEDURE.md

```diff
@@ -0,0 +1,49 @@
+c-ares release procedure - how to do a release
+==============================================
+
+in the source code repo
+-----------------------
+
+- edit `RELEASE-NOTES` to be accurate
+
+- make sure all relevant changes are committed on the master branch
+
+- tag the git repo in this style: `git tag -a cares-1_13_0` -a annotates the
+  tag and we use underscores instead of dots in the version number.
+
+- run "./maketgz 1.13.0" to build the release tarball. It is important that
+  you run this on a machine with the correct set of autotools etc installed
+  as this is what then will be shipped and used by most users on *nix like
+  systems.
+
+- push the git commits and the new tag
+
+- gpg sign the tarball
+
+- upload the resulting files to https://c-ares.haxx.se/download/
```

Comment:
Would it help to say the tarball must be signed with a GPG key signed by at least two commiters to c-ares? That way, any person 'trusted enough' would be permitted to release (of course, barring some emergencies, it would be you..)